### ISSUE TITLE
.gitkeep warning on macOS

### ISSUE THREAD

**Issue by prusnak** [trezor/trezord-go#171]
<img width="914" alt="ihhkkck19b731" src="https://user-images.githubusercontent.com/42201/60388602-68875080-9ab4-11e9-985f-4b227f52b1cf.png">

**Comment by darwin** [comment]
This `.gitkeep` is likely choking uninstaller (issue #186).

Today I was trying to uninstall v2.0.27 on macOS 10.15.7

Look for `Error relinking file (primary): /Library/InstallerSandboxes/.PKInstallSandboxManager/92D98A4B-AF84-4D5A-8633-4023D70F831F.activeSandbox/Root/.gitkeep to /.gitkeep, error = 30`

```
> cat /var/log/system.log
...
2020-10-24 23:11:50+02 bismarck15 Installer[58420]: ================================================================================
2020-10-24 23:11:50+02 bismarck15 Installer[58420]: User picked Standard Install
2020-10-24 23:11:50+02 bismarck15 Installer[58420]: Choices selected for installation:
2020-10-24 23:11:50+02 bismarck15 Installer[58420]: 	Upgrade: "TREZOR Bridge"
2020-10-24 23:11:50+02 bismarck15 Installer[58420]: 	Upgrade: "Uninstall TREZOR Bridge"
2020-10-24 23:11:50+02 bismarck15 Installer[58420]: 		uninstall.pkg#uninstall.pkg : com.bitcointrezor.pkg.TREZORBridge.uninstall : 2.0.27
2020-10-24 23:11:50+02 bismarck15 Installer[58420]: ================================================================================
2020-10-24 23:11:50+02 bismarck15 Installer[58420]: It took 0.00 seconds to summarize the package selections.
2020-10-24 23:11:50+02 bismarck15 Installer[58420]: -[IFDInstallController(Private) _buildInstallPlanReturningError:]: location = file://localhost
2020-10-24 23:11:50+02 bismarck15 Installer[58420]: -[IFDInstallController(Private) _buildInstallPlanReturningError:]: file://localhost/Applications/Utilities/TREZOR%20Bridge/uninstall.pkg#uninstall.pkg
2020-10-24 23:11:50+02 bismarck15 Installer[58420]: Set authorization level to root for session
2020-10-24 23:11:53+02 bismarck15 Installer[58420]: Administrator authorization granted.
2020-10-24 23:11:53+02 bismarck15 Installer[58420]: Will use PK session
2020-10-24 23:11:53+02 bismarck15 Installer[58420]: Using authorization level of root for IFPKInstallElement
2020-10-24 23:11:53+02 bismarck15 Installer[58420]: Starting installation:
2020-10-24 23:11:53+02 bismarck15 Installer[58420]: Configuring volume "BismarckOS"
2020-10-24 23:11:53+02 bismarck15 Installer[58420]: Preparing disk for local booted install.
2020-10-24 23:11:53+02 bismarck15 Installer[58420]: Free space on "BismarckOS": 65,91 GB (65911783424 bytes).
2020-10-24 23:11:53+02 bismarck15 Installer[58420]: Create temporary directory "/var/folders/j9/cwjmkvc523sfc1pz6gt1x10m0000gn/T//Install.58420bIb03j"
2020-10-24 23:11:53+02 bismarck15 Installer[58420]: IFPKInstallElement (1 packages)
2020-10-24 23:11:53+02 bismarck15 Installer[58420]: Current Path: /System/Library/CoreServices/Installer.app/Contents/MacOS/Installer
2020-10-24 23:11:53+02 bismarck15 installd[44454]: PackageKit: Adding client PKInstallDaemonClient pid=58420, uid=501 (/System/Library/CoreServices/Installer.app/Contents/MacOS/Installer)
2020-10-24 23:11:53+02 bismarck15 Installer[58420]: PackageKit: Enqueuing install with framework-specified quality of service (utility)
2020-10-24 23:11:53+02 bismarck15 installd[44454]: PackageKit: ----- Begin install -----
2020-10-24 23:11:53+02 bismarck15 installd[44454]: PackageKit: request=PKInstallRequest <1 packages, destination=/>
2020-10-24 23:11:53+02 bismarck15 installd[44454]: PackageKit: packages=(
	    "PKLeopardPackage <id=com.bitcointrezor.pkg.TREZORBridge.uninstall, version=2.0.27, url=file:///Applications/Utilities/TREZOR%20Bridge/uninstall.pkg#uninstall.pkg>"
	)
2020-10-24 23:11:53+02 bismarck15 installd[44454]: PackageKit: Set reponsibility for install to 58420
2020-10-24 23:11:53+02 bismarck15 installd[44454]: PackageKit: Will do receipt-based obsoleting for package identifier com.bitcointrezor.pkg.TREZORBridge.uninstall (prefix path=)
2020-10-24 23:11:53+02 bismarck15 installd[44454]: PackageKit: Extracting file:///Applications/Utilities/TREZOR%20Bridge/uninstall.pkg#uninstall.pkg (destination=/Library/InstallerSandboxes/.PKInstallSandboxManager/92D98A4B-AF84-4D5A-8633-4023D70F831F.activeSandbox/Root, uid=0)
2020-10-24 23:11:53+02 bismarck15 installd[44454]: PackageKit: prevent user idle system sleep
2020-10-24 23:11:53+02 bismarck15 installd[44454]: PackageKit: suspending backupd
2020-10-24 23:11:53+02 bismarck15 installd[44454]: PackageKit: Using trashcan path /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/PKInstallSandboxTrash/92D98A4B-AF84-4D5A-8633-4023D70F831F.sandboxTrash for sandbox /Library/InstallerSandboxes/.PKInstallSandboxManager/92D98A4B-AF84-4D5A-8633-4023D70F831F.activeSandbox
2020-10-24 23:11:53+02 bismarck15 installd[44454]: PackageKit: Shoving /Library/InstallerSandboxes/.PKInstallSandboxManager/92D98A4B-AF84-4D5A-8633-4023D70F831F.activeSandbox/Root (1 items) to /
2020-10-24 23:11:53+02 bismarck15 install_monitor[58432]: Temporarily excluding: /Applications, /Library, /System, /bin, /private, /sbin, /usr
2020-10-24 23:11:53+02 bismarck15 shove[58433]: Error relinking file (primary): /Library/InstallerSandboxes/.PKInstallSandboxManager/92D98A4B-AF84-4D5A-8633-4023D70F831F.activeSandbox/Root/.gitkeep to /.gitkeep, error = 30
2020-10-24 23:11:53+02 bismarck15 shove[58433]: srcPath = /Library/InstallerSandboxes/.PKInstallSandboxManager/92D98A4B-AF84-4D5A-8633-4023D70F831F.activeSandbox/Root/.gitkeep NSFileOwnerAccountID=0 NSFileHFSTypeCode=0 NSFileSystemFileNumber=52765665 NSFileExtensionHidden=0 NSFileSystemNumber=16777222 NSFileSize=0 NSFileGroupOwnerAccountID=80 NSFileHFSCreatorCode=0 NSFileCreationDate=2019-05-13 11:58:15 +0000 NSFileOwnerAccountName=root NSFilePosixPermissions=420 NSFileType=NSFileTypeRegular NSFileGroupOwnerAccountName=admin NSFileReferenceCount=1 NSFileModificationDate=2019-05-13 11:58:15 +0000
2020-10-24 23:11:53+02 bismarck15 shove[58433]: dstParentPath = / NSFileOwnerAccountID=0 NSFileSystemFileNumber=2 NSFileExtensionHidden=0 NSFileSystemNumber=16777222 NSFileSize=704 NSFileGroupOwnerAccountID=80 NSFileOwnerAccountName=root NSFileCreationDate=2019-10-11 11:48:25 +0000 NSFilePosixPermissions=493 NSFileType=NSFileTypeDirectory NSFileGroupOwnerAccountName=admin NSFileReferenceCount=22 NSFileModificationDate=2020-09-27 07:11:00 +0000
2020-10-24 23:11:53+02 bismarck15 shove[58433]: [src=unknown] /.gitkeep: unable to restore flags to destination (error 2)
2020-10-24 23:11:53+02 bismarck15 shove[58433]: [source=file] failed _RelinkFile(/Library/InstallerSandboxes/.PKInstallSandboxManager/92D98A4B-AF84-4D5A-8633-4023D70F831F.activeSandbox/Root/.gitkeep, /.gitkeep): Read-only file system
2020-10-24 23:11:53+02 bismarck15 shove[58433]: srcPath = /Library/InstallerSandboxes/.PKInstallSandboxManager/92D98A4B-AF84-4D5A-8633-4023D70F831F.activeSandbox/Root/.gitkeep NSFileOwnerAccountID=0 NSFileHFSTypeCode=0 NSFileSystemFileNumber=52765665 NSFileExtensionHidden=0 NSFileSystemNumber=16777222 NSFileSize=0 NSFileGroupOwnerAccountID=80 NSFileHFSCreatorCode=0 NSFileCreationDate=2019-05-13 11:58:15 +0000 NSFileOwnerAccountName=root NSFilePosixPermissions=420 NSFileType=NSFileTypeRegular NSFileGroupOwnerAccountName=admin NSFileReferenceCount=1 NSFileModificationDate=2019-05-13 11:58:15 +0000
2020-10-24 23:11:53+02 bismarck15 shove[58433]: dstPath = /.gitkeep does not exist
2020-10-24 23:11:53+02 bismarck15 shove[58433]: dstParentPath = / NSFileOwnerAccountID=0 NSFileSystemFileNumber=2 NSFileExtensionHidden=0 NSFileSystemNumber=16777222 NSFileSize=704 NSFileGroupOwnerAccountID=80 NSFileOwnerAccountName=root NSFileCreationDate=2019-10-11 11:48:25 +0000 NSFilePosixPermissions=493 NSFileType=NSFileTypeDirectory NSFileGroupOwnerAccountName=admin NSFileReferenceCount=22 NSFileModificationDate=2020-09-27 07:11:00 +0000
2020-10-24 23:11:53+02 bismarck15 install_monitor[58432]: Re-included: /Applications, /Library, /System, /bin, /private, /sbin, /usr
2020-10-24 23:11:53+02 bismarck15 installd[44454]: PackageKit: releasing backupd
2020-10-24 23:11:53+02 bismarck15 installd[44454]: PackageKit: allow user idle system sleep
2020-10-24 23:11:53+02 bismarck15 installd[44454]: PackageKit: Cleared responsibility for install from 58420.
2020-10-24 23:11:53+02 bismarck15 installd[44454]: PackageKit: Cleared permissions on Installer.app
2020-10-24 23:11:53+02 bismarck15 installd[44454]: PackageKit: Install Failed: Error Domain=PKInstallErrorDomain Code=120 "An unexpected error occurred while moving files to the final destination." UserInfo={NSLocalizedDescription=An unexpected error occurred while moving files to the final destination., arguments=(
	    "-f",
	    "-s",
	    "/Library/InstallerSandboxes/.PKInstallSandboxManager/92D98A4B-AF84-4D5A-8633-4023D70F831F.activeSandbox/Root",
	    "/"
	), NSUnderlyingError=0x7ff143d10490 {Error Domain=NSPOSIXErrorDomain Code=30 "Read-only file system"}} {
	    NSLocalizedDescription = "An unexpected error occurred while moving files to the final destination.";
	    NSUnderlyingError = "Error Domain=NSPOSIXErrorDomain Code=30 \"Read-only file system\"";
	    arguments =     (
	        "-f",
	        "-s",
	        "/Library/InstallerSandboxes/.PKInstallSandboxManager/92D98A4B-AF84-4D5A-8633-4023D70F831F.activeSandbox/Root",
	        "/"
	    );
	}
2020-10-24 23:11:53+02 bismarck15 installd[44454]: PackageKit: Running idle tasks
2020-10-24 23:11:53+02 bismarck15 Installer[58420]: install:didFailWithError:Error Domain=PKInstallErrorDomain Code=120 "An unexpected error occurred while moving files to the final destination." UserInfo={NSUnderlyingError=0x600001634270 {Error Domain=NSPOSIXErrorDomain Code=30 "Read-only file system"}, NSLocalizedDescription=An unexpected error occurred while moving files to the final destination., arguments=(
	    "-f",
	    "-s",
	    "/Library/InstallerSandboxes/.PKInstallSandboxManager/92D98A4B-AF84-4D5A-8633-4023D70F831F.activeSandbox/Root",
	    "/"
	)}
2020-10-24 23:11:53+02 bismarck15 installd[44454]: PackageKit: Removing client PKInstallDaemonClient pid=58420, uid=501 (/System/Library/CoreServices/Installer.app/Contents/MacOS/Installer)
2020-10-24 23:11:53+02 bismarck15 installd[44454]: PackageKit: Done with sandbox removals
2020-10-24 23:11:54+02 bismarck15 Installer[58420]: Install failed: The Installer encountered an error that caused the installation to fail. Contact the software manufacturer for assistance.
2020-10-24 23:11:54+02 bismarck15 Installer[58420]: IFDInstallController 18913E0 state = 8
2020-10-24 23:11:54+02 bismarck15 Installer[58420]: Displaying 'Install Failed' UI.
2020-10-24 23:11:54+02 bismarck15 Installer[58420]: 'Install Failed' UI displayed message:'The Installer encountered an error that caused the installation to fail. Contact the software manufacturer for assistance.'.
```

**Comment by prusnak** [comment]
Fixed in 80a5abfe7c6dc3ad327389c77daa6e71bcebe88b
